schema/spacewalk/common/tables/endpoint.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- Copyright (c) 2025 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE access.endpoint (
+    id              NUMERIC NOT NULL PRIMARY KEY,
```

Comment:
We should not use NUMERIC anymore. Also no sequence, but do not forget a constraint
Use:

BIGINT CONSTRAINT suse_issper_id_pk PRIMARY KEY
                          GENERATED ALWAYS AS IDENTITY